lib/rule-tester/rule-tester.js

```diff
@@ -477,13 +481,13 @@ class RuleTester {
                  * Assumes everything on the item is a config except for the
                  * parameters used by this tester
                  */
-                const itemConfig = lodash.omit(item, RuleTesterParameters);
+                const itemConfig = omit(RuleTesterParameters, item);
```

Comment:
Not as straightforward, but lets us skip a barely-used dependency
```suggestion
                const itemConfig = {...item};
                for (const parameter of RuleTesterParameters) {
                	delete[parameter];
                }
```

Reply:
Resolved in ada00d52f.